packages/meditrak-server/src/routes/editUser.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
```

Comment:
Even inside meditrak-server we now use the generic mono-repo header for new files

(I only notice because of your updates to the others)